src/sql/src/plan/statement/dml.rs

```diff
@@ -474,7 +467,10 @@ pub fn plan_explain_schema(
 
 pub fn plan_explain_timestamp(
     scx: &StatementContext,
-    ExplainTimestampStatement { format, query }: ExplainTimestampStatement<Aug>,
+    ExplainTimestampStatement {
+        format,
+        select: query,
```

Comment:
nit: mabye you can rename `query` to `select` here already?

Reply:
Done.